app/src/main/java/org/schabi/newpipe/player/PlayQueueActivity.java

```diff
@@ -506,6 +509,10 @@ private void openPlaylistAppendDialog(final List<PlayQueueItem> playlist) {
             () -> PlaylistCreationDialog.newInstance(d).show(getSupportFragmentManager(), TAG));
     }
 
+    private void clearQueue() {
+        player.getPlayQueue().removeAll();
+    }
```

Comment:
```suggestion
    private void clearQueue() {
        player.onPlaybackShutdown();
    }
```
After playing with this all day, I have found this works pretty well.  I have not found any bugs after running the code like this, and the style checks out.  The only problem, currently, is the bottom player still persists.  Working on that, now.